client/tsconfig.json

```diff
@@ -20,6 +20,6 @@
     "noUnusedParameters": true,
     "noFallthroughCasesInSwitch": true
   },
-  "include": ["src", "../model/classes"],
+  "include": ["src", "../backend/model/classes"],
```

Comment:
I don't think this addition is needed anymore. We can remove this and just keep `src`

Reply:
Do you just mean remove "../backend/model/classes"?